src/en/clean-copy/03-Part II. The Globe of Belgium/15.md

```diff
@@ -1,65 +1,65 @@
 
-Finally, what monks-experimenters we're talking about? Monasteries were obliged to be self-sustainable — implying growing oranges locally, which looks quite unrealistic in Belgian climate — to say nothing about cities and guilds, not monks, being the main drivers of innovation in brewing in the 15th-century Netherlands.
+Finally, what monks or experimentalosts are we talking about? Monasteries by their social make-up were obliged to be self-sustainable. This implies growing oranges locally which would be quite unrealistic in a Belgian climate. This does not even consider that it would have been city merchants and guilds, not monks, who would have been the main drivers of innovation in brewing within the 15th-century Netherlands.
 
-The answer to all these questions is quite simple: though Pierre Celis actually got a recipe of authentic Hoegaarden beer from Loius Tomsin himself, the newly made *witbier* had nothing to do with it: the Tomsin recipe (and the earlier ones as well) contains no oranges or coriander, but does contain oats (though let's be honest, Celis' Hoegarden did include a small proportion of oats until 80s)[ref:mulder-lost-beers:"When Hoegaarden was still spontaneously fermented"](https://lostbeers.com/when-hoegaarden-was-still-spontaneously-fermented/).
+The answer to all these questions is quite simple.  Though Pierre Celis actually procured a recipe of authentic Hoegaarden beer from Loius Tomsin himself, the newly made *witbier* had nothing to do with the original *classical* conception. The Tomsin recipe (nor the earlier ones as well) contains no oranges or coriander, but does contain oats (though let's be honest, Celis's White beer conception did include a small proportion of oats until the 1980's)[ref:mulder-lost-beers:"When Hoegaarden was still spontaneously fermented"](https://lostbeers.com/when-hoegaarden-was-still-spontaneously-fermented/).
 
-In 1985, Celis sold the Hoegaarden brand to the Artois company (now AB InBev) and moved to the US, where he opened a new brewery borrowing his name — the Celis Brewery. American consumers got a taste for *witbier*, and soon other brewers start to make it — above-mentioned Allagash, also Ommegang, Samuel Adams, Bell's, Canadian Unibroue, and even the mainstream giant, MillerCoors (under the Blue Moon brand).
+In 1985, Celis sold the Hoegaarden brand to the Artois company (now AB InBev) and moved to the US, where he opened a new brewery under his own name — the Celis Brewery. American consumers began developing a taste for *witbier*, and soon other brewers started to make it — the above-mentioned Allagash, also Ommegang, Samuel Adams, Bell's, Canadian Unibroue,the mainstream giant MillerCoors (under the Blue Moon brand), as well as a host of smaller community micro brewers.
```

Comment:
I'd replaced it with just 'and others'. The 'micro brewery' term will be introduced much later in the Part III.

Reply:
I removed micro and left community brewers. 'Others' is a vague term. People can visualize their local pub. Up to you, my suggestion.